code/game/objects/structures/ghost_role_spawners.dm

```diff
@@ -487,10 +487,8 @@
 	death = FALSE
 	random = TRUE
 	mob_species = /datum/species/human
-	flavour_text = "<span class='big bold'>You are a security officer working for Nanotrasen,</span><b> stationed onboard a state of the art research station. You vaguely recall rushing into a \
-	cryogenics pod due to an oncoming radiation storm. The last thing you remember is the station's Artificial Program telling you that you would only be asleep for eight hours. As you open \
-	your eyes, everything seems rusted and broken, a dark feeling swells in your gut as you climb out of your pod. \
-	Work as a team with your fellow survivors and do not abandon them.</b>"
+	short_desc = "You are a security officer working for Nanotrasen, stationed onboard a state of the art research station."
+	flavour_text = "You vaguely recall rushing into a cryogenics pod due to an oncoming radiation storm. The last thing you remember is the station's Artificial Program telling you that you would only be asleep for eight hours. As you open your eyes, everything seems rusted and broken, a dark feeling swells in your gut as you climb out of your pod. Work as a team with your fellow survivors and do not abandon them."
```

Comment:
It'd be nice if it was across multiple lines rather than 1 long line like
```
flavour_text = "ahierth oarsiht aosrhitg oahsdfg asfg \
asoighaosrihg toasirhg oaisdhfg oaidfhg \
asriotghoadfhigao ihdrg asdr fg."
```

Reply:
Will do when I get home in a couple of hours

Reply:
done